src/guides/bioinformatics/filtering-and-subsampling.rst

```diff
@@ -86,6 +86,25 @@ total sequences:
      --output-sequences subsampled_sequences.fasta \
      --output-metadata subsampled_metadata.tsv
 
+.. note::
+
+   ``--subsample-max-sequences`` works by internally calculating a value for
+   ``--sequences-per-group``.
+
+.. note::
+
+   For these options, the number of targeted sequences per group does not take
+   into account:
+
+   1. The actual number of sequences available in the input data. For example,
+      consider a dataset with 200 sequences available from 2023 and 100
+      sequences available from 2024. ``--group-by year --subsample-max-sequences
+      300`` is equivalent to ``--group-by year --sequences-per-group 150``. This
+      will take 150 sequences from 2023 and all 100 sequences from 2024 for a
+      total of 250 sequences, which is less than the target of 300.
+   2. Any sequences force-included by ``--include`` or ``--include-where``. This
+      may result in higher than targeted sequences for some groups.
```

Comment:
For me, this brings up the question of how does subsampling interact with `--exclude` or `--exclude-where` options. Can it result in lower than targeted sequences?

Reply:
[Adding details on order of operations](https://github.com/nextstrain/docs.nextstrain.org/pull/222#discussion_r1720073616) should address this.

Reply:
Addressed with 1b7e25f72719eee1e00de29b1c245a459533212d